src/containers/donation/Donation.jsx

```diff
@@ -0,0 +1,272 @@
+                That’s over 4,000 people a year, making suicide a leading cause
+                of death in Canada. We can help prevent suicide with the right
+                services. A donation to the Canada Suicide Prevention Service
+                allows us to connect individuals in distress and individuals
+                with suicidal thoughts with local Crisis Centres to receive
+                immediate support.
+                <br />
+                Approximately 11 people iight services. A donation to the Canada
+                Suicide Prevention Service allows us to connect individuals in
+                distress and individuals with suicidal thoughts with local
+                Crisis Centres to receive immediate support.
+              </p>
+            </section>
+          </Col>
+        </Row>
+
+        <Row>
+          <Col xs={12} sm={12} md={12} lg={12}>
+            <div className="wholeCarousel">
+              <Carousel>
+                <Carousel.Item class="carousel-indicators">
+                  <img
+                    className="d-block w-100"
+                    src="https://i.ibb.co/4p24Dd4/Quotes2-in-Donation-Page.png"
+                    alt="First slide"
+                  />
+                </Carousel.Item>
+
+                <Carousel.Item>
+                  <img
+                    className="d-block w-100"
+                    src="https://i.ibb.co/4p24Dd4/Quotes2-in-Donation-Page.png"
+                    alt="second slide"
+                  />
+                </Carousel.Item>
+
+                <Carousel.Item>
+                  <img
+                    className="d-block w-100 quotes"
+                    src="https://i.ibb.co/4p24Dd4/Quotes2-in-Donation-Page.png"
+                    alt="Third slide"
+                  />
+                </Carousel.Item>
+              </Carousel>
+            </div>
+          </Col>
+        </Row>
+
+        <Row>
+          <section className="pageMarings">
+            <p className="textBody1">
+              Approximately 11 people in Canada die by suicide every day. That’s
+              over 4,000 people a year, making suicide a leading cause of death
+              in Canada. We can help prevent suicide with the right services. A
+              donation to the Canada Suicide Prevention Service allows us to
+              connect individuals in distress and individuals with suicidal
+              thoughts with local Crisis Centres to receive immediate support.
+              <br />
+              Approximately 11 people iight services. A donation to the Canada
+              Suicide Prevention Service allows us to connect individuals in
+              distress and individuals with suicidal thoughts with local Crisis
+              Centres to receive immediate support.
+            </p>
+          </section>
+        </Row>
+
+        <Row>
+          <div className="socialMedia">
+            <a href="http://www.facebook.com">
```

Comment:
here you can use bootstrap nav component
https://react-bootstrap.github.io/components/navs/

Reply:
@basil-kawak I think it will not make a big difference, won't it?